tests/ctst/features/replicatateExistingObjects.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Replicate existing objects created before enabling replication
+    This feature tests the script crrExistingObjects.js from S3utils that replicates
+    objects created before the replication was enabled on source the bucket.
+
+    @2.7.0
+    @PreMerge
+    @ReplicationTest
+    Scenario Outline: Object created before adding the replication setting is replicated with the script
+        Given an existing bucket "<sourceBucket>" "with" versioning, "without" ObjectLock "without" retention mode
+        And an object "<sourceObject>" that "exists"
+        And a replication configuration from "<sourceBucket>" bucket to "<replicationLocation>" location
+        When I run the job to replicate existing objects with status "<sourceObjStatus>" from bucket "<sourceBucket>" to location "<replicationLocation>"
+        Then the object "<sourceObject>" should be replicated within 60 seconds from "<sourceBucket>" bucket
+        And the replicated object "<sourceObject>" from "<sourceBucket>" bucket to "<replicationBucket>" bucket with location "<replicationLocation>" should be the same as its source object
+
+        Examples:
+            | sourceBucket  | sourceObjStatus |                    replicationBucket |    sourceObject |       replicationLocation |
+            | source-bucket |             NEW | ci-zenko-aws-replication-ctst-bucket | source-object-1 | awsbackendreplicationctst |
```

Comment:
Replicating "existing" objects is just one use for crrExistingObjects.

There is however another use case we will want to test: it can also be used to trigger replication of objects which failed to replicate.
It may be done in a separate PR and/or ticket though, as making the replication fail (in the test setup) may be complicated...

(one way to do that may be by setting bucket policy that prevents writing in the bucket, then setting up live replication and waiting
until the object has indeed failed to replicate; then we can remove bucket policy and run crrExistingObjects, and finally watch as it works)